liquidprompt

```diff
@@ -2343,6 +2348,28 @@ _lp_runtime_color() {
     lp_runtime_color="${LP_COLOR_RUNTIME}${lp_runtime_format}${NO_COL}"
 }
 
+######################
+# Kubernetes context #
+######################
```

Comment:
Don't add a new section for this, add it to the Development Environment section.